k8s/openebs-operator.yaml

```diff
@@ -45,7 +45,7 @@ rules:
   resources: ["customresourcedefinitions"]
   verbs: [ "get", "list", "create", "update", "delete", "patch"]
 - apiGroups: ["*"]
-  resources: [ "disks", "blockdevices", "blockdeviceclaims"]
+  resources: ["blockdevices", "blockdeviceclaims"]
```

Comment:
@akhilerm -- let us retain the option to `disks`. And probably => have some code in the upcoming or this release, that actually deletes the older disks?

Reply:
@kmova  But we removed the controller for disk, and deleted the APIs.

Reply:
yep.. as part of m-apiserver upgrade, can we clean up old stale entries or CRDs. were there any finalizers on disk resources?

Reply:
If we don't do this, then customers might have disk entries that need to be manually cleaned up.. right?

Reply:
> were there any finalizers on disk resources?

No

Reply:
> If we don't do this, then customers might have disk entries that need to be manually cleaned up.. right?

Yes. Entries will be there. But in that case. It would have been better if we removed entries as part of reconciliation and then in another release, remove the controller.

Reply:
yes, we missed to the above. As we move from 1.x releases of openebs into 2.0, we will have to have users first upgrade to the last stable release of 1.x. In which we need to catch and cleanup all these things. When we start tagging 2.0, we can remove all the deprecated resources.

Reply:
But for "catching up" those disk resources, we need the API right? We removed it in this release (1.10).

Reply:
Done.